lib/services/platform-service.ts

```diff
@@ -236,13 +236,21 @@ export class PlatformService extends EventEmitter implements IPlatformService {
 			platform = this.$mobileHelper.normalizePlatformName(platform);
 			this.$logger.trace("Validate options for platform: " + platform);
 			const platformData = this.$platformsData.getPlatformData(platform, projectData);
-			return await platformData.platformProjectService.validateOptions(projectData.projectId, provision, teamId);
+			return await platformData.platformProjectService.validateOptions(
+				projectData.projectIdentifiers[platform.toLowerCase()],
+				provision,
```

Comment:
a bit strange to pass `provision` in a common file like `platform-service.ts` when it's iOS specific

Reply:
Will check it, but will probably fix in separate PR.